lms/templates/courseware/progress.html

```diff
@@ -105,6 +105,18 @@ <h2 class="title">${_("Congratulations, you qualified for a certificate!")}</h2>
           </div>
           %endif
 
+          %if certificate_invalidated:
```

Comment:
Just realized this blob is outside of the `%if show_generate_cert_btn:`. And you are duplicating the DOM structure here. We should move this block into the `%if show_generate_cert_btn:` statement. I am looking to refactor the whole block in ECOM-4681